pymc3/backends/smc_text.py

```diff
@@ -303,8 +303,6 @@ def create_result_trace(self, stage_number=-1, idxs=(-1, ), step=None, model=Non
         ----------
         stage_number : int
             stage of which result traces are loaded
```

Comment:
If you remove idxs here they should be removed in the function input as well. However, I would prefer to have it flexible. Simply put in the smc.py when create_result_trace() is executed idxs=range(n_steps) rather than making it unflexible here. Then you can revert all the changes here in create_result_trace. What do the others think? @ColCarroll @twiecki ?

Reply:
@hvasbath thanks for the comments. I think your suggestion is very reasonable. I will make those changes. About the recycling idea I am not sure about that now (but I keep reading/thinking) I think using all samples from the last stage is a good alternative. I also realized that to compute the marginal likelihood (evidence) we just need to make small changes to SMC, today I was testing that and is working :-) Not shure if I should add that as part of this PR or as a separated one.

Reply:
I think it is better to put it in a seperate PR.